spec/checker_spec.lua

```diff
@@ -55,20 +55,21 @@ describe("Scope analysis: ", function()
             "variable 'x' is not declared")
     end)
 
-    it("forbids multiple toplevel declarations with the same name", function()
+    it("forbids multiple toplevel declarations with the same name for exported functions", function()
         assert_error([[
-            local function f() end
-            local function f() end
+            export function f() end
+            export function f() end
         ]],
-            "duplicate toplevel declaration for 'f'")
+            "duplicate export 'f', previous one at line 1")
```

Comment:
Could you please add another test case for an exported function that has the same name as an exported variable? Just so that we don't forget to add that test case later. You can use `pending()` for now.